site/src/App/routes/examples/marketing-banner/marketing-banner.tsx

```diff
@@ -0,0 +1,655 @@
+                    <Button weight="weak">Show me</Button>
+                  </Inline>
+                </Stack>
+              </Column>
+              <Column>
+                <Placeholder height={300} label="Marketing illustration" />
+              </Column>
+            </Columns>
+          </Box>
+        </Step>
+
+        <Text component="p">
+          Now if you click the &ldquo;Open in Playroom&rsquo; button on the code
+          block above, our layout scaling up pretty well from mobile through to
+          desktop. However, you&rsquo;ll notice on large screens the content
+          spans the full width.
+        </Text>
+
+        <Text component="p">Let&rsquo;s tackle this next.</Text>
+
+        <Step
+          heading="9. Constraining larger layouts"
+          detail={
+            <Stack space="large">
+              <Text component="p">
+                To provide consistent structure and aid readability, most pages
+                constrain layouts to a maximum width—we achieve this by wrapping
+                the content in a{' '}
+                <TextLink href="/components/ContentBlock">
+                  ContentBlock
+                </TextLink>
+                .
+              </Text>
+              <Text component="p" tone="secondary">
+                Note: We want to ensure that the coloured box is not constrained
+                and extends full width, so be sure to add the `ContentBlock`
+                inside the `Box` but around the `Columns`.
+              </Text>
+            </Stack>
+          }
+        >
+          <Box background="brandAccent" paddingX="small" paddingY="xlarge">
+            <ContentBlock>
+              <Columns space="gutter" alignY="center">
+                <Column width="3/5">
+                  <Stack space="large">
+                    <Heading level="1">
+                      Heard about our latest marketing campaign?
+                    </Heading>
+                    <Inline space="none">
+                      <Button weight="weak">Show me</Button>
+                    </Inline>
+                  </Stack>
+                </Column>
+                <Column>
+                  <Placeholder height={300} label="Marketing illustration" />
+                </Column>
+              </Columns>
+            </ContentBlock>
+          </Box>
+        </Step>
+
+        <Step
+          heading="10. Polish!"
+          detail={
+            <Stack space="large">
+              <Text component="p">
+                Now for some final spacing polish we can also leverage the same
+                responsive technique to finesse the spacing across screen sizes.
+              </Text>
+              <Text component="p">
+                As an example let&rsquo;s increase the vertical padding on the
+                `Box` to &ldquo;xxlarge&rdquo; above mobile by setting
+                `paddingY` to [ &ldquo;xlarge&rdquo;, &ldquo;xxlarge&rdquo; ]
```

Comment:
Noticing that we're inconsistent on spaces inside of responsive props. Happy to go either way, but I think we should be consistent.